src/simple.jl

```diff
@@ -3,7 +3,7 @@
 
     SimpleTimes(len::Integer, duration::Vector{T}) where {T}
     SimpleTimes(len::Integer, duration::Number)
-    SimpleTimes(dur::Vector{T}) where {T<:Number}
+    SimpleTimes(dur::Vector{T}) where {T}
```

Comment:
Is it relevant to allow for durations that are not numbers? Do you want to have the duration as a parameter?

Reply:
Not sure, I just went all the way reasoning who am I to decide what can be a parameter. But that could be going too far, maybe.

Reply:
It is actually hard for me to say where the problem lies in this situation as I do not know the practical implementation.